README.md

```diff
@@ -215,6 +215,28 @@ fmt.Println(issues, githubError, resp, err)
 
 Pass a nil `successV` or `failureV` argument to skip JSON decoding into that value.
 
+### HTTP Tracing
```

Comment:
### Modify a Request

Sling provides the raw `http.Request` so modifications can be made using standard `net/http` features. For example, in Go 1.7+ , add [HTTP tracing](link) to a request with a context:

(example)